src/action/finder/glob_to_regex.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020 Google LLC
+//
+// Use of this source code is governed by an MIT-style license that can be found
+// in the LICENSE file or at https://opensource.org/licenses/MIT.
+
+use crate::session::RegexParseError;
+use regex::Regex;
+
+/// Converts glob expression into a Rust Regex.
+/// E.g. "asd*[!123]??" will be converted into "asd.*[^123]..".
```

Comment:
Prefer to use examples that are verified when you run tests, e.g. like [here][1] or [here][2].

[1]: https://github.com/google/rrg/blob/40447d159b1d65ff0a7a5056503509a780a4aba1/src/metadata.rs#L58-L85
[2]: https://github.com/google/rrg/blob/40447d159b1d65ff0a7a5056503509a780a4aba1/src/fs/mod.rs#L34-L77

Reply:
Added an example. I agree it's much better that way.